apps/nextjs/src/stores/lessonPlanStore/stateActionFunctions/handleMessagesUpdated.ts

```diff
@@ -0,0 +1,109 @@
+import {
```

Comment:
This file is the important part of this store. I probably need a few more comments to explain what it's doing...